app/workers/deployment_watchdog.rb

```diff
@@ -0,0 +1,21 @@
+class DeploymentWatchdogWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: 'deployment_watchdog'
+  def perform(auto_deployment_id)
+    auto_deployment = AutoDeployment.find_one(id: auto_deployment_id)
```

Comment:
Little more conventional to just do:

```ruby
AutoDeployment.find(auto_deployment_id)
```

Reply:
@ejholmes can I still use `find_one` ?

Reply:
You can use it. In almost all Rails codebases (including this one) you'll see `.find(id)`, so I'd prefer consistency/convention if we can.